src/Promenade.Web/Views/Shared/_Layout.cshtml

```diff
@@ -140,15 +152,28 @@
                                                 @foreach (var subItem in item.Navigations)
                                                 {
                                                     <li>
-                                                        <a href="@subItem.Link"
-                                                           class="dropdown-item"
-                                                           aria-label="@subItem.NavigationText.AriaLabel">
-                                                            @if (!string.IsNullOrEmpty(subItem.Icon))
-                                                            {
+                                                        @if (string.IsNullOrEmpty(subItem.NavigationText.Label)
+                                                            && !string.IsNullOrEmpty(subItem.Icon)
+                                                            && !string.IsNullOrEmpty(subItem.NavigationText.Title))
+                                                        {
+                                                            <a href="@subItem.Link"
+                                                               class="dropdown-item"
+                                                               title="@subItem.NavigationText.Title">
                                                                 <span class="@subItem.Icon"></span>
-                                                            }
-                                                            @subItem.NavigationText.Label
-                                                        </a>
+                                                                @subItem.NavigationText.Label
```

Comment:
This label text should be safe to remove since it'll be empty.

Reply:
Resolved in commit 9b9fd46116bc0d3399fe1289b3f3852d4fd5576c